app/assets/javascripts/components/overview/details.jsx

```diff
@@ -127,7 +128,9 @@ const Details = createReactClass({
 
   poll() {
     return setInterval(() => {
-      if (this.state.updateCount > MAX_UPDATE_COUNT) { return; }
+      const documentVisible = document.hidden;
+
+      if (documentVisible || (this.state.updateCount > MAX_UPDATE_COUNT)) { return; }
```

Comment:
Oh, I see, it's part of the guard clause, so it will return without re-fetching if the document is not focused. In that case, maybe rename it `documentNotFocused` or `documentHidden`?